packages/fixie-common/src/client.ts

```diff
@@ -100,6 +100,9 @@ export class FixieClientBase {
 
   async requestJson<T = Jsonifiable>(path: string, bodyData?: unknown, method?: string): Promise<T> {
     const response = await this.request(path, bodyData, method);
+    if (response.status === 204) {
```

Comment:
Should this be checking `Content-Type` instead?

Reply:
I don't think so. 204 has no content-type, right?